src/ops/ops.go

```diff
@@ -568,6 +569,29 @@ func (o *ops) CleanRaidDevice(device string) error {
 	return nil
 }
 
+func (o *ops) GetRaidDevices(deviceName string) ([]string, error) {
```

Comment:
The distinction between getRaidDevices and GetRaidDevices is not immediately clear from the function names, one has to read the code to tell the difference. The former should be something along the lines of `getAllRaidDevices` and the latter should maybe be `getDeviceRaidDevices`. The latter could really use some documentation to explain its purpose though, because I don't think there's a good name to explain it well enough

Reply:
please note that only one of them is public and should used externally but i will add documentation